api/apps/geoprocessing/src/modules/planning-area/planning-area.dto.ts

```diff
@@ -1,10 +1,31 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { GeoJSON } from 'geojson';
+import { BBox, GeoJSON } from 'geojson';
+import { isDefined } from '@marxan/utils';
+import { PuSizes } from './pu-sizes';
+import { MarxanGeoJson } from './marxan-geo-json';
 
 export class PlanningAreaDto {
   @ApiProperty()
   id!: string;
 
   @ApiProperty()
   data!: GeoJSON;
+
+  static from(input: {
+    id: string;
+    data: GeoJSON;
+    bbox?: BBox;
+    puSizes?: PuSizes;
+  }) {
+    const marxanGeoJSON = new MarxanGeoJson(input.data);
+    if (isDefined(input.bbox)) marxanGeoJSON.addBBox(input.bbox);
```

Comment:
each `GeoJSON` must have a `bbox` already. How we do ensure that bbox is actually a bbox around the shapes inside?

Reply:
hmm, I have no idea why I assumed it's optional.

we do not validate it, we just trust what we got from the DB

Reply:
fixed